packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -36,6 +36,28 @@ const Color _kLightThemeSplashColor = const Color(0x66C8C8C8);
 const Color _kDarkThemeHighlightColor = const Color(0x40CCCCCC);
 const Color _kDarkThemeSplashColor = const Color(0x40CCCCCC);
 
+/// Configures the tap target and layout size of certain Material widgets.
+///
+/// The following widgets are impacted:
+///
+///   * [FloatingActionButton], only the mini tap target size is increased.
+///   * [MaterialButton]
+///   * [OutlineButton]
+///   * [FlatButton]
+///   * [TimePicker]
+///   * [SnackBar]
```

Comment:
at least RawMaterialButton and RawChip are missing from this list. Maybe it should just say something like "Some of the affected widgets include" or something?

Reply:
Fixed, though technically RawMaterialButton isn't effected since it specifically doesn't use ThemeData directly.